docs/tutorials/complete_applications/complete_application/step_03/ReadMe.md

```diff
@@ -132,64 +95,26 @@ max_capacity_cfg = Config.configure_data_node(id="max_capacity", default_data=20
 ### Remaining Data Nodes
 
 - *cleaned_dataset* is the dataset after cleaning (after the `clean_data()` function).
-  with a `scope.GLOBAL`. It means if the initial dataset didn't change, Taipy will not re-execute the `clean_data()` 
-  task. In other words, after the creation of this data node through `clean_data()`, Taipy knows that it is not 
-  necessary to create it again.
 
 - *predictions* are the predictions of the model. In this pipeline, it will be the output of the `predict_baseline()` 
   function. Each pipeline will create its own *prediction* Data Node hence `scope=Scope.PIPELINE`.
 
 ```python
 ## Remaining Data Nodes
 cleaned_dataset_cfg = Config.configure_data_node(id="cleaned_dataset",
-                                             validity_period=dt.timedelta(days=1),
-                                             scope=Scope.GLOBAL) 
-
-predictions_cfg = Config.configure_data_node(id="predictions", scope=Scope.PIPELINE)
-```
-
+                                                 scope=Scope.GLOBAL) 
 
-## Functions
-
-Here’s the code of each of the two *Python* functions: `clean_data()` and `predict_baseline()`. Their goal is 
-respectively to clean the data and to predict the data.
-
-```python
-def clean_data(initial_dataset: pd.DataFrame):
-    print("     Cleaning data")
-    # Convert the date column to datetime
-    initial_dataset["Date"] = pd.to_datetime(initial_dataset["Date"])
-    cleaned_dataset = initial_dataset.copy()
-    return cleaned_dataset
-
-
-def predict_baseline(cleaned_dataset: pd.DataFrame, n_predictions: int, day: dt.datetime, max_capacity: int):
-    print("     Predicting baseline")
-    # Select the train data
-    train_dataset = cleaned_dataset[cleaned_dataset["Date"] < day]
-    
-    predictions = train_dataset["Value"][-n_predictions:].reset_index(drop=True)
-    predictions = predictions.apply(lambda x: min(x, max_capacity))
-    return predictions
+predictions_cfg = Config.configure_data_node(id="predictions")
 ```
 
-## Tasks
-
-Tasks are the translation of functions in Taipy. These tasks combined with Data Nodes create your graph (DAG). 
-Creating a task is simple; you need:
-
-- An id
+## Task Configuration
 
-- A function
+Tasks are the translation of functions in Taipy. Each task has an ID, a function, inputs, and outputs.
 
-- Inputs
-
-- Outputs
 
 ### clean_data_task
 
-The first task that you want to create is your `clean_data()` task. It will take your initial dataset (input Data 
-Node), clean it (calling the `clean_data()` function) and generate the cleaned dataset Data Node.
+The first task that you want to create is your `clean_data()` task. It will take your initial dataset (input Data Node), clean it (calling the `clean_data()` function) and generate the cleaned dataset Data Node. This task will only execute once thanks to the skippability feature of Taipy.
```

Comment:
```suggestion
The first task that you want to create is your `clean_data()` task. It will take your initial
dataset (input Data Node), clean it (calling the `clean_data()` function) and generate the 
cleaned dataset Data Node. This task will only execute once thanks to the skippability feature of Taipy.
```